{{cookiecutter.repo_name}}/pyproject.toml

```diff
@@ -57,10 +57,6 @@ requires = [
 
 [tool.setuptools-git-versioning]
 enabled = true
-version_file = "VERSION"
-count_commits_from_version_file = true
-dev_template = "{tag}.dev{ccount}"
-dirty_template = "{tag}.dev{ccount}"
```

Comment:
Why remove this?  

This ensures that a version number is found by setuptools-git-versioning, even if there is no tag

Reply:
The `version_file` switched from tag-based to file-based versioning and I was under the impression that we preferred tag-based over file-based. Do you have the preference to use file-based?

Also had some issues with the pyproject.toml not being parseable by a number of tools. The issue seemed to be connected to this somehow but considering the previous paragraph, I didn't really look into it so not sure.

Reply:
The ```version_file``` is needed in case the branch doesn't contain any tags.  Without this,  setuptools_git_versioning can't determine a version number and causes an error during package build/install (also for editable install)  with a pretty obscure and unhelpful error message.  This file is just a fallback for when no tag is found. 

Note that I have seen some issues with the ```dev/dirty_template```.  This is caused if you don't have any commits on the branch/tag.  ccount becomes zero which can cause an issue.  I think this is a bug in setuptools_git_versioning.

Reply:
Replaced with starting_version thanks to >= 2.0.0 setuptools-git-versioning